src/twisted/web/http.py

```diff
@@ -1909,6 +1910,7 @@ def __init__(
         self.finishCallback = finishCallback
         self._buffer = bytearray()
         self._start = 0
+        self._trailerHeaders = []
```

Comment:
It looks like MyPy is [failing](
https://github.com/twisted/twisted/actions/runs/6276966953/job/17068965951?pr=11998) for want of an annotation here:

```
src/twisted/web/http.py:1913:9: error: Need type annotation for "_trailerHeaders" (hint: "_trailerHeaders: List[<type>] = ...")  [var-annotated]
```